<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>486</width>
     <height>230</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-3</y>
       <width>489</width>
       <height>205</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>489</width>
        <height>205</height>
       </rect>
      </property>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>150</y>
         <width>469</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="SplitMixed">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>18</y>
         <width>470</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Split mixed output</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="DefaultOutputFolder">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>45</y>
         <width>470</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Default export folder</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBoxExport">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>80</y>
         <width>468</width>
         <height>60</height>
        </rect>
       </property>
       <property name="title">
        <string>Output Directory</string>
       </property>
       <layout class="QHBoxLayout" name="ExportLayout">
        <item>
         <widget class="QLineEdit" name="outExportDirLine"/>
        </item>
        <item>
         <widget class="QPushButton" name="outExportDirBrowseBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="ExportButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>178</y>
         <width>105</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QPushButton" name="OkButton">
       <property name="geometry">
        <rect>
         <x>397</x>
         <y>178</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFilesProcessed">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>183</y>
         <width>279</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QCheckBox" name="GenerateBlankBackSubscans">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>15</y>
       <width>469</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate blank background subscans</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="KeepOriginalColorIllumForeSubscans">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>42</y>
       <width>468</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Keep the original color and illumination in foreground subscans (lengthy)</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
